docs/Displaying a User Interface/Alerts/index.md

```diff
@@ -1,135 +1,217 @@
 # Alerts
 An alert is a pop-up window that can show a short message with optional buttons. When an alert is activated, it will abort any SDL operation that is in-progress, except the already-in-progress alert. If an alert is issued while another alert is still in progress, the newest alert will simply be ignored.
 
+Depending the platform, an alert can have up to three lines of text, a progress indicator (e.g. a spinning wheel or hourglass), and up to four soft buttons.
+
 !!! NOTE
 The alert will persist on the screen until the timeout has elapsed, or the user dismisses the alert by selecting a button. There is no way to dismiss the alert programmatically other than to set the timeout length.
 !!!
 
-## Alert UI
-Depending the platform, an alert can have up to three lines of text, a progress indicator (e.g. a spinning wheel or hourglass), and up to four soft buttons.
+## Creating an Alert
+### Alert With No Soft Buttons
+!!! NOTE
+If no soft buttons are added to an alert some OEMs may add a default "cancel" or "close" button.
+!!!
 
-### Alert HMI
-###### Generic HMI
+#### Alert HMI
 ![Generic - Alert](assets/Generic_alert.png)
 
-###### Ford HMI
-![SYNC 3 - Alert](assets/SYNC3_alert.jpg)
+@![iOS]
+#####  Objective-C
```

Comment:
Remove these and use them for the "Creating the Alert" part below.

Reply:
Let me know if how I have it know is good after the other fixes